debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/infinispan/marshalling/LobWriteEventAdapter.java

```diff
@@ -63,4 +65,26 @@ public LobWriteEvent factory(int eventType, String scn, String tableId, String r
     public String getData(LobWriteEvent event) {
         return event.getData();
     }
+
+    /**
+     * A ProtoStream handler to extract the {@code offset} field from a {@link LobWriteEvent} type.
+     *
+     * @param event the event instance, must not be {@code null}
+     * @return the offset of the data to be written for a LOB field
+     */
+    @ProtoField(number = 8, required = true)
```

Comment:
Marking `getOffset` and `getLength` as `required` will mean that the code isn't backward compatible and older caches won't be loadable.  That's likely fine since this is all still incubating; so we'll need to include a migration note about this.

Reply:
I can also make them optional, and add some logic to deal with missing values in TransactionCommitConsumer. That will complicate the code solely for the purpose of dealing with this transition case, though, so I'm not sure if adding the migration note isn't the more pragmatic solution.
Up to you - if you want me to deal with optional length/offset fields, I can figure out a way to fit that in.

Reply:
Nah; Infinispan support was added in 1.7 and it's still technically "incubating" so I think it's fine to require a re-snapshot.  I mostly made the note here in the PR so we don't lose sight of the migration note needed.  I'll add something to the issue right now so it's where we need it for release purposes.